cmd/experimental/kueue-viz/frontend/start.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
+
+# Generate env.js with runtime environment variables
+cat << EOF > ./build/env.js
+/*
```

Comment:
Could you please move it after copyright prefix?

Reply:
There is trick here: I am generating the start.sh and also adding the copyright header to it.

I will had the copyright also in start.sh then.

Reply:
Ah, I see :)